### PR TITLE
Fix ArcGIS Feature request failures on specific server

### PR DESCRIPTION
**Pull request by mdastous-bentley**
This PR solves 2 issues:

1. Capabilities parsing issue of the `supportedQueryFormats`:
  Typically, the capabilities response contains the following:
  `"supportedQueryFormats": "JSON, PBF",`
  Turns out some server, do no include a whitespace after the comma separator, like this:
   `"supportedQueryFormats": "JSON,PBF",`

3. The `f` (i.e. format) query parameter, must have its value set in lower case, even though the `supportedQueryFormats` capability indicates the formats in upper case.
This is confirmed by the [ArcGIS documentation](https://developers.arcgis.com/rest/services-reference/enterprise/query-feature-service-layer/):
image

### CODE REVIEW

File: extensions/map-layers-formats/src/ArcGisFeature/ArcGisFeatureProvider.ts [hunk 1]
```diff
@@ -215,11 +215,11 @@ export class ArcGisFeatureProvider extends ArcGISImageryProvider {
     // Note: needs to be checked on the layer metadata, service metadata advertises a different set of formats
     //       Also, since PBF format does not support floating points, there is no point using this format if supportsCoordinatesQuantization is not available.
     if (this._layerMetadata.supportedQueryFormats) {
-      const formats: string[] = this._layerMetadata.supportedQueryFormats.split(", ");
-      if (formats.includes("PBF") && this._supportsCoordinatesQuantization) {
-        this._format = "PBF";
-      } else if (formats.includes("JSON")) {
-        this._format = "JSON";
+      const formats: string[] = this._layerMetadata.supportedQueryFormats.split(",").map((s: string) => s.trim().toLowerCase());
```
Comment (aruniverse) [inline extensions/map-layers-formats/src/ArcGisFeature/ArcGisFeatureProvider.ts:218]: why not trim before split?

**Review by aruniverse — APPROVED** [review]

File: extensions/map-layers-formats/src/ArcGisFeature/ArcGisFeatureProvider.ts (hunk 1 above)
Comment (tcobbs-bentley) [reply extensions/map-layers-formats/src/ArcGisFeature/ArcGisFeatureProvider.ts:218]: This way trims each comma-separated component. Trimming first wouldn't do that.

**Review by tcobbs-bentley — APPROVED** [review]

**Comment by mdastous-bentley** [comment]
@Mergifyio backport release/4.9.x

**Review by eringram — APPROVED** [review]

**Comment by mdastous-bentley** [comment]
@aruniverse Looks like Im getting random test failures ?

**Comment by aruniverse** [comment]
> @aruniverse Looks like Im getting random test failures ?

Yes integration tests are failing due to some auth issues, we are aware and looking into it

**Pull request merged**